src/components/coop/coop.styles.js

```diff
@@ -1,7 +1,111 @@
 import styled from 'styled-components';
+import { fontSize, lineHeight, space } from "styled-system";
 
 export const Container = styled.div`
+max-width: 100%;
+    margin: 60px 0 100px 0;
+    width: 100%;
+    display: grid;
+    grid-gap: 60px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 1fr;
 
+
+    @media only screen and (max-width: 1024px) {
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(2, 1fr);
+    }
+`;
+
+export const CardContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Card = styled.div`
+    border-radius: 50px;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    position: relative;
+    overflow: hidden;
+    
+    &:after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background: rgba(0, 0, 0, 0.5);
+    }
+
+    @media only screen and (max-width: 1024px) {
+        width: 446px;
+        height: 326px;   
+    }
 `;
+    
+
+
+export const Image = styled.img`
+    src: ${props => props.src};
+`;
+
+export const Title = styled.h2`
```

Comment:
Can you use the GlobalSubtitle or the TitleComponent to create this one? like the example I showed you